musicbrainz_bot/editing.py

```diff
@@ -311,6 +311,33 @@ def add_area(self, area: dict, edit_note: str, auto=False) -> str:
 
         return self._extract_mbid("area")
 
+    def edit_area(
+        self, gid: str, area: dict, update: dict, edit_note: str, auto=False
+    ) -> str:
+        """Posts an edit for an existing area based on existing data and update data.
+
+        Args:
+            area (dict): Existing data for the area to be edited. Including gid
+            update (dict): Updated data for the area to be edited. Follows the same structure as the area dict in add_area
+            edit_note (str): edit note
+            auto (bool, optional): Marks if an edit is 'votable' or 'auto-edit'. Defaults to False.
+
+        Returns:
+            str: returns the area MBID of the edited area.
+        """
+        update.update({"edit_note": edit_note})
```

Comment:
I might be missing something, but what's the benefit of passing edit note separately and then adding it to `update` here? Is it to keep the dict structure?

Reply:
Well, that's because the in every other function in this bot, the edit note is generally passed as a parameter to the edit function, and not in the data dictionary itself.

I thought following the convention might be a good idea :))

Reply:
We can easily turn the function around to receive the edit note in the dict itself if you wish.

Reply:
Ok, I guess so :)